EasyCommands/Common/Operations.cs

```diff
@@ -89,11 +89,11 @@ partial class Program {
             AddBiOperation<bool, bool>(BiOperand.COMPARE, (a, b) => a.CompareTo(b));
             AddBiOperation<string, string>(BiOperand.COMPARE, (a, b) => a.CompareTo(b));
             AddBiOperation<float, float>(BiOperand.COMPARE, (a, b) => a.CompareTo(b));
-            AddBiOperation<Vector3D, Vector3D>(BiOperand.COMPARE, (a, b) => a.Equals(b));
+            AddBiOperation<Vector3D, Vector3D>(BiOperand.COMPARE, (a, b) => !a.Equals(b));
             AddBiOperation<Color, Color>(BiOperand.COMPARE, (a, b) => a.PackedValue.CompareTo(b.PackedValue));
             AddBiOperation<Vector3D, float>(BiOperand.COMPARE, (a, b) => a.Length().CompareTo(b));
             AddBiOperation<float, Vector3D>(BiOperand.COMPARE, (a, b) => a.CompareTo(b.Length()));
-            AddBiOperation<KeyedList, KeyedList>(BiOperand.COMPARE, (a, b) => Enumerable.SequenceEqual(a.keyedValues, b.keyedValues));
+            AddBiOperation<KeyedList, KeyedList>(BiOperand.COMPARE, (a, b) => !Enumerable.SequenceEqual(a.keyedValues, b.keyedValues));
```

Comment:
This doesn't seem right to me.  I don't understand how "myList is empty" continues to work with this change.

I ran tests locally after changing these and the tests failed for me.  Not sure what's going on here.  

We should add SimpleComparisonTests to test all the expected comparisons, similar to what we did for casting.  

Let me know if you don't really want to take that on and I can try to tackle it.

Reply:
`Compare` should return  `less/-1`, `equal/0` or `greater/1` and is only really welldefined for types that can be ordered, like numbers or strings (lexographic).
Neither `vector` nor `list` can be ordered, they can only  be checked for equality.
Atleast to my knowledge it is customary to return `greater/1` for  inequality in this case.

So simply negating the the equality is shorter than `expr ? 0 : 1` to restore the proper integer values, because the `BiOperation`s wrap the result into an `Primitive` that needs to be unpacked on `Compare` (via a cast to a number) anyway.
This only works because the proper `number` to `bool` (and vice versa) cast behaviour is implemented now.

Reply:
To you want  the tests in a seperate PR or in here? I can take a look at that over the weekend.

Reply:
Let's go ahead and add them here just for completeness.  Thank you for tackling this one!